docs/data_entry_flow_index.md

```diff
@@ -343,9 +343,47 @@ class TestFlow(config_entries.ConfigFlow, domain=DOMAIN):
         return self.async_create_entry(title="Some title", data={})
 ```
 
+### Show Menu
+
+This will show a navigation menu to the user to easily pick the next step. The menu labels can be hardcoded by specifying a dictionary of {`step_id`: `label`} or translated via `strings.json` when specifying a list.
+
+```python
+class ExampleConfigFlow(data_entry_flow.FlowHandler):
+    async def async_step_user(self, user_input=None):
+        return self.async_show_menu(
+            step_id="user",
+            menu_options=["discovery", "manual"],
+            description_placeholders={
+                "model": "Example model",
+            }
+        )
+        return self.async_show_menu(
```

Comment:
This has two return values, so its not valid python i think. Is that intentional? I realize it show two different approaches.

Reply:
Ah yes, it's illustrative, let me add a comment.
```suggestion
        # Example showing the other approach
        return self.async_show_menu(
```